VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionUtils.java

```diff
@@ -0,0 +1,38 @@
+package org.edx.mobile.discussion;
+
+import android.app.Activity;
+import android.support.v4.widget.TextViewCompat;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import org.edx.mobile.R;
+import org.edx.mobile.third_party.iconify.IconDrawable;
+import org.edx.mobile.third_party.iconify.Iconify;
+
+public abstract class DiscussionUtils {
+    /**
+     * Sets the state, text and icon of the new item creation button on discussion screens
+     *
+     * @param activity  The current activity
+     * @param isTopicClosed Boolean if the topic is closed or not
+     * @param textView The TextView whose text has to be updated
+     * @param txtResourceId The text resource to be applied to the textView
+     * @param icon The icon to set
+     * @param creationLayout The layout which should be enabled/disabled and applied listener to
```

Comment:
I think its in the right place, cuz its followed by a listener object at the end which is what I've mostly seen in SDK's function prototypes.